webapp/src/app/shared/widgets/langlabel-widget/langlabel-widget.component.html

```diff
@@ -0,0 +1,41 @@
+<div fxLayout="row" class="property-container" [matTooltip]="attribute?.description">
+	<label [for]="key" class="property-name">{{key}}:</label>
```

Comment:
there is not element with the id `key` set, so the `for` attribute shouldn't be set here